operators/README.md

```diff
@@ -79,6 +80,14 @@ go build ./cmd/laboratory-operator/main.go
 
 N.B. So far, the readiness check for VirtualMachines is performed by assuming that the operator is running on the same cluster of the Virtual Machines. This prevents the possibility to have *ready* VMs when testing the operator outside the cluster.
 
+To test the persistent feature, after the creation of the instance you need to manually add the annotation
```

Comment:
1) this sentence is split across two lines
2) Please rephrase the sentence, because it looks convoluted.

Reply:
Actually we need to remove this sentence, because now we use vm and not vmi with a different logic, so it is deprecated